README.md

```diff
@@ -1,18 +1,39 @@
-[![Coverage Status](https://coveralls.io/repos/github/TeamStoic/StoicChess/badge.svg?branch=master)](https://coveralls.io/github/TeamStoic/StoicChess?branch=master)
+==========
+StoicChess
+==========
 
-[ ![Codeship Status for TeamStoic/StoicChess](https://codeship.com/projects/8443e920-1f83-0134-0385-3ae309bf0e43/status?branch=master)](https://codeship.com/projects/160373)
+[![Codeship Status for TeamStoic/StoicChess](https://codeship.com/projects/8443e920-1f83-0134-0385-3ae309bf0e43/status?branch=master)](https://codeship.com/projects/160373) [![Coverage Status](https://coveralls.io/repos/github/TeamStoic/StoicChess/badge.svg?branch=master)](https://coveralls.io/github/TeamStoic/StoicChess?branch=master)
```

Comment:
IMO the badges should be Above the main title. Also, we should think about what license this is to be released under & add a badge for that as well